integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioTwoDomainsInImage.java

```diff
@@ -296,7 +296,7 @@ public void testIstioTwoDomainsWithSingleIngress() {
     checkConsole = checkAppUsingHostHeader(consoleUrl, domainNamespace2 + ".org");
     assertTrue(checkConsole, "Failed to access domain2 WebLogic console");
     logger.info("WebLogic console on domain2 is accessible");
-    result = DeployUtil.deployUsingRest(K8S_NODEPORT_HOST, 
+    result = DeployUtil.deployToClusterUsingRest(K8S_NODEPORT_HOST, 
```

Comment:
We usually use static import. Static import DeployUtil.deployToClusterUsingRest and change this DeployUtil.deployToClusterUsingRest to deployToClusterUsingRest.

Reply:
done